wazimap_ng/profile/views.py

```diff
@@ -114,8 +114,16 @@ def get(self, request, profile_id):
 
 @api_view()
 def profile_geography_indicator_data(request, profile_id, geography_code, profile_indicator_id):
+    version_name = request.GET.get('version', None)
     profile = get_object_or_404(models.Profile, pk=profile_id)
-    geography = get_object_or_404(Geography, code=geography_code)
+    if version_name is None:
+        version_name = profile.geography_hierarchy.default_version
```

Comment:
Same question as the previous for version_name

`version_name = request.GET.get('version', profile.geography_hierarchy.default_version)`

Reply:
Updated